docs/user-guide/map-views.md

```diff
@@ -0,0 +1,64 @@
+# Map Views
+
+The **Map Views** is a [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) tool useful to configure multiple 3D views and switch between them with navigation tool.
```

Comment:
It works also in 2D even if it has less options, I don't know if we want mention this